docs/tutorials/customizing-the-output/add-custom-files-directories-in-custom-locations.md

```diff
@@ -239,4 +241,4 @@ def getServiceName(filePath):
 {% endraw %}
 ```
 
-Next step [Customize in-built transformer behavior](/tutorials/customizing-the-output/customize-inbuilt-transformer-behavior)
\ No newline at end of file
+Next step [Customizing generated Dockerfile and behavior of inbuilt transformer](/tutorials/customizing-the-output/custom-dockerfile-change-inbuilt-behavior)
```

Comment:
This is currently the last transformer. So it is not supposed to have the next step. We can remove the line.
```suggestion
```